src/common/config.ts

```diff
@@ -1,6 +1,16 @@
 
-function getEnv() {
-  return process.env['NODE_ENV'];
+function environmentVariables() {
+  const NODE_ENV: string = process.env['NODE_ENV'] || 'local';
+  const DB_HOSTNAME: string = process.env['NODE_ENV'] || 'localhost';
```

Comment:
Not the DB hostname, but [os.hostname()](https://nodejs.org/api/os.html#os_os_hostname).

Reply:
Oh. I was wondering about whether the OS Hostname or the DB hostname. Will fix!